src/components/NewBook.js

```diff
@@ -1,26 +1,50 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+// import PropTypes from 'prop-types';
```

Comment:
- [ Required ] I think it would be better if you remove any piece of commented code from your code base. Commented code makes your project unprofessional and is also considered a sign of code that is not clean. Please consider rectifying this.